test/unit/testUtil.js

```diff
@@ -21,34 +16,6 @@ define('testUtil', ['bluebird', 'narrativeConfig', 'json!/test/testConfig.json']
         return userId;
     }
 
-    function getCurrentNarrative() {
-        if (currentNarrative) {
-            return currentNarrative;
-        }
-        const token = getAuthToken();
-        // make a new workspace
-        // make a new narrative object and add it to the workspace
-        // add some data to the workspace
-    }
```

Comment:
Unused, and currentNarrative was always null, so it was not a particularly useful function.